examples/task.json

```diff
@@ -8,29 +8,17 @@
 	"workflow":{
 		"collect":{
 			"metrics":{
-				"/intel/cpp/mock/*":{}
+				"/intel/cpp/mock/rando/int32":{},
+				"/intel/cpp/mock/rando/string":{},
+				"/intel/cpp/mock/rando/int64":{},
+				"/intel/cpp/mock/rando/boolean":{},
+				"/intel/cpp/mock/dynamic/*/string":{}
 			},
 			"config":{
 				"/intel/cpp/mock/randomnumber/one":{
 					"password":"silly goose"
 				}
-			},
```

Comment:
Why are you removing section for processor and publisher from exemplary task manifest? It is part of helpful example which is described in https://github.com/intelsdi-x/snap-plugin-lib-cpp/tree/master/examples#snap-plugin-c-library-examples,  this task manifest is used in `run.sh` script.